packages/usa-accordion/src/styles/_usa-accordion.scss

```
@@ -30,20 +44,38 @@ $accordion-border: units($theme-accordion-border-width) solid
   width: 100%;
 
   &:hover {
-    background-color: color("base-lighter");
-    color: color("ink");
+    @include set-text-and-bg(
+      $accordion-button-background-active-color,
+      $context: $accordion-context
+    );
+    @include set-icon-from-bg(
+      $accordion-button-background-active-color,
+      "usa-icons/remove",
+      "usa-icons-bg/remove--white"
+    );
     text-decoration: none;
   }
 
   @media (forced-colors: active) {
     forced-color-adjust: none;
   }
 }
-// scss-lint:enable PropertyCount
 
 @mixin accordion-button-unopened-styles {
-  @include add-background-svg("usa-icons/add");
+  @include set-icon-from-bg(
+    $theme-accordion-button-background-color,
+    "usa-icons/add",
+    "usa-icons-bg/add--white"
+  );
   background-size: units(3);
+
+  &:hover {
+    @include set-icon-from-bg(
+      $accordion-button-background-active-color,
+      "usa-icons/add",
+      "usa-icons/add--white"
```

I received the error `Error: Can't resolve '../uswds-core/src/styles/mixins/img/usa-icons/add--white.svg'` when setting `$theme-accordion-button-background-color` to `"green-40".

```suggestion
      "usa-icons-bg/add--white"
```